src/DecryptionFailureTracker.ts

```diff
@@ -140,32 +189,61 @@ export class DecryptionFailureTracker {
     //     localStorage.setItem('mx-decryption-failure-event-ids', JSON.stringify([...this.trackedEvents]));
     // }
 
-    public eventDecrypted(e: MatrixEvent): void {
+    /** Callback for when an event is decrypted.
+     *
+     * This function should be called after a decryption attempt on an event.  It
+     * should be called whether the decryption is successful or not.
```

Comment:
As an alternative to making this private:

```suggestion
    /** Callback for when an event is decrypted, successfully or otherwise.
     *
     * It is unnecessary to call this function explicitly: it is automatically called on a decryption event.
     * It is only exposed for testing.
```

(I'd probably vote in favour of the ts-ignore dance for unit tests. You could probably even encapsulate it in a single helper method in the test. But at this point I think we should just land this rather than attempting to gold-plate it.)